src/blockchain_db/blockchain_db.h

```diff
@@ -160,15 +160,14 @@ struct txpool_tx_meta_t
   uint64_t max_used_block_height;
   uint64_t last_failed_height;
   uint64_t receive_time;
-  uint64_t last_relayed_time;
+  uint64_t last_relayed_time; //!< If received over i2p/tor, randomized forward time. Otherwise, last relayed timestamp
   // 112 bytes
-  uint8_t kept_by_block;
```

Comment:
Seems reasonable to introduce 
```c++
bool kept_by_block() const noexcept;
```

Reply:
And grep well since anything that does "if (foo.kept_by_block)" will be turned into "if (true)".

Reply:
I will definitely make this change, provided the DB change to `master` is acceptable.

Reply:
Didn't see any objections (or any comments at all), so this function was added and used where appropriate.